litestar/cli/_utils.py

```diff
@@ -252,7 +252,7 @@ def wrapped(ctx: Context, /, *args: P.args, **kwargs: P.kwargs) -> T:
 
 def _wrap_commands(commands: Iterable[Command]) -> None:
     for command in commands:
-        if isinstance(command, Group):
```

Comment:
This test failed after upgrading rich-click: https://github.com/litestar-org/litestar/blob/56b7395fa75a304dfea5711fe62646c469ce8616/tests/unit/test_cli/test_cli.py#L68

Issue is that, even though we import `Group` from `click`, it is actually `rich_click.patch._PatchedRichGroup`, and where `command` is an instance of `rich_click.rich_command.RichGroup`, `isinstance(command, Group)` returns `False`, meaning that the inner commands of groups weren't being wrapped to inject args.

This change seems to work, but I'm not familiar enough with click to know if there is a better way. @litestar-org/maintainers

Reply:
Seems like an okay fix 👍